developers/index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/html">
+<head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-167873271-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+        function gtag(){dataLayer.push(arguments);}
+        gtag('js', new Date());
+        gtag('config', 'UA-167873271-1');
+    </script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="ScholarX is an exclusive 6-month program aimed at providing
+                                      mentoring support to a selected pool of high Potential undergraduate students
+                                      based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of
+                                      the world’s top universities or Fortune 500 companies. It’s our free premium
+                                      mentoring platform by Sri Lankans for Sri Lankans working towards creating a culture
+                                      of knowledge and expertise sharing without the limitation of geographical borders.">
```

Comment:
meta data needs to be chaned right?

Reply:
Oh shit, I forgot, I'll update it 
Thank you for pointing it out @anjula-sack

Reply:
Resolved with the commit 6cc6fe1
Please review @anjula-sack